<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cusplot</class>
 <widget class="QMainWindow" name="cusplot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cusplot</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QGroupBox" name="groupBox_cusplot">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QGridLayout" name="gridLayout_2"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="tracer">
             <property name="text">
              <string>tracer ON/OFF</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="vl_params_all">
            <item>
             <layout class="QHBoxLayout" name="hl_param_a">
              <item>
               <widget class="QLabel" name="label_sin_a">
                <property name="text">
                 <string>Параметр а</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_a">
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="hl_param_b">
              <item>
               <widget class="QLabel" name="label_sin_b">
                <property name="text">
                 <string>Параметр b</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_b">
                <property name="text">
                 <string>10</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="hl_param_c">
              <item>
               <widget class="QLabel" name="label_sin_c">
                <property name="text">
                 <string>Параметр c</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_c">
                <property name="text">
                 <string>0.1</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="hl_param_e">
              <item>
               <widget class="QLabel" name="label_sin_e">
                <property name="text">
                 <string>Параметр e</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_e">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="vL_start_buttoms">
            <item>
             <widget class="QPushButton" name="ReadFileReal">
              <property name="text">
               <string>Read real file</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ReadFileReeGo">
              <property name="text">
               <string>Read Reego file</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="startButton">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Построить sin</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="createPDF">
              <property name="text">
               <string>Отчёт</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hl_graphic_buttoms">
          <item>
           <layout class="QVBoxLayout" name="vl_gb_1">
            <item>
             <widget class="QPushButton" name="source">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Исходный</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="linear_Regression">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>LR</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Linear_Least_Squares">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>LLSQ</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="vl_gb_2">
            <item>
             <widget class="QPushButton" name="medianFilter">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Median</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="smooth">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Скользящее</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="interpolation">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Сплайн</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="vl_gb_3">
            <item>
             <widget class="QPushButton" name="merge">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Экстремумы</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sigma">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Sigma</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRead"/>
    <addaction name="actionWrite"/>
    <addaction name="actionPrint_Dir"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRead"/>
   <addaction name="actionWrite"/>
   <addaction name="actionPrint_Dir"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRead">
   <property name="text">
    <string>Read</string>
   </property>
  </action>
  <action name="actionWrite">
   <property name="text">
    <string>Write</string>
   </property>
  </action>
  <action name="actionPrint_Dir">
   <property name="text">
    <string>Print Dir</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
